subprojects/language-java/src/integTest/groovy/org/gradle/javadoc/JavadocIntegrationTest.groovy

```diff
@@ -61,6 +62,30 @@ class JavadocIntegrationTest extends AbstractIntegrationSpec {
         file("build/docs/javadoc/Foo.html").text.contains("""Hey Joe!""")
     }
 
+    @Issue("gradle/gradle#1090")
+    @Unroll
+    def "allow single quote characters in options when string delimited with #delimiter"() {
+        given:
+        buildFile << """
+            apply plugin: "java"
+            javadoc.options.header = ${delimiter}${header}${delimiter}
+        """
+
+        file("src/main/java/Foo.java") << "public class Foo {}"
+
+        when:
+        succeeds 'javadoc'
+
+        then:
+        file("build/docs/javadoc/Foo.html").text.contains('"\'header\'"')
+
+        where:
+        delimiter | header
```

Comment:
Thank you. Sorry about getting confused and dragging it out. I've only been using github for a couple of months, trying to learn and pick up some new skills. My mistakes here are a reminder I need to work on my communication skills as much as my programming and testing ones. Thanks again!